src/SDK/MonitoringApplianceEmulator/Controls/ChartPropertyControl.xaml

```diff
@@ -13,6 +13,9 @@
             <lvc:CartesianChart.AxisX>
                 <lvc:Axis LabelFormatter="{Binding XAxisFormatter}"></lvc:Axis>
             </lvc:CartesianChart.AxisX>
+            <lvc:CartesianChart.AxisY>
+                <lvc:Axis LabelFormatter="{Binding YAxisFormatter}"></lvc:Axis>
+            </lvc:CartesianChart.AxisY>
```

Comment:
please attach screenshot

Reply:
![image](https://user-images.githubusercontent.com/6997688/49340159-adddbf80-f644-11e8-8faf-a8aad3caf1e5.png)

Reply:
is there a way to limit the Y axis so it won't show negative numbers if there aren't any in the data?
can you open a bug on this please?

---
In reply to: [238098178](https://github.com/Azure/SmartDetectorsMonitoringAppliance/pull/31#discussion_r238098178) [](ancestors = 238098178)